ui/src/tasks/components/TasksHeader.tsx

```diff
@@ -5,21 +5,19 @@ import {Page} from 'src/pageLayout'
 // Components
 import {SlideToggle, ComponentSize} from '@influxdata/clockface'
 import {Tabs, ComponentSpacer, Alignment, Stack} from 'src/clockface'
-import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
 import TaskOrgDropdown from 'src/tasks/components/TasksOrgDropdown'
 import AddResourceDropdown from 'src/shared/components/AddResourceDropdown'
 
 import 'src/tasks/components/TasksPage.scss'
 
 interface Props {
   onCreateTask: () => void
-  setSearchTerm: (searchTerm: string) => void
   setShowInactive: () => void
   showInactive: boolean
   toggleOverlay: () => void
   showOrgDropdown?: boolean
   isFullPage?: boolean
-  searchTerm: string
+  filterComponent: () => JSX.Element
```

Comment:
I ended up doing this instead of conditionally rendering the search component within the header so that there is less prop drilling. Unrelated but I started using render props in the `ResourceList.Card` component to great effect and wanted to use the pattern more elsewhere.

As far as being a function that returns a component vs the component itself, @chnn recommended I do it this way.